### DISCUSSION TITLE
Is 1.19.4 coming?

### DISCUSSION THREAD

**Discussion by ilciab** [BlossomMods/BlossomLib#6] [General]
Just wanted to know if and when the mod will be updated to 1.19.4

**Comment by CodedSakura** [comment]
it depends on https://github.com/NucleoidMC/Server-Translations which does not have a 1.19.4 release yet, sadly

**Reply by CodedSakura** [reply]
it does seem like the pre-releases for 1.19.4 of Server Translations do make BlossomLib work on 1.19.4, but it does not feel like a long-term solution at all

**Reply by CodedSakura** [reply]
(you can get the pre-release here https://github.com/NucleoidMC/Server-Translations/releases/tag/v2.0.0-beta.2%2B1.19.4-pre2)

**Reply by ilciab** [reply]
Thank you for replying, I think I’ll wait for a stable release of the mods though

**Comment by CodedSakura** [comment]
i have now released a 1.19.4 version using the Server-Translations pre-release, as it seems stable enough

**Reply by ilciab** [reply]
Thank you now I will be able to upgrade my server to 1.19.4
